README.md

```diff
@@ -370,35 +370,19 @@ restApiConfig:
 
 ## Step 9: Customize Model Deployment
 
-In the `ecsModels` section of `config.yaml`, configure the models you want to deploy.
+In the `ecsModels` section of `config.yaml`, allow our deployment process to pull the model weights for you.
 
-The configuration file will determine which models are deployed. In order to deploy an additional model or a
-different model the only required change is to the configuration file, as long as it is compatible with the inference
-container. Specifically, see the `ecsModels` section of the [config.yaml](./config.yaml) file.
-Here we define the model name, if we want to deploy, the type of instance we want to deploy to, the type of model
-(textgen or embedding), the inference container and then the containerConfig. There are many more parameters for the
-ecs models, many for autoscaling and health checks. However, let's focus on the model specific ones:
+During the deployment proces, lisa will optionally attempt to download your model weights if you specify an optional`ecsModels`
+array, this will only work in non ADC regions. Specifically, see the `ecsModels` section of the [config.yaml](./config.yaml) file.
+Here we define the model name, inference container, and baseImage:
 
 ```yaml
 ecsModels:
   - modelName: your-model-name
-    deploy: false
-    instanceType: g4dn.12xlarge
-    modelType: textgen
     inferenceContainer: tgi
-    containerConfig:
-      baseImage: ghcr.io/huggingface/text-generation-inference:1.0.2
-      environment:
-        QUANTIZE: bitsandbytes-nf4
-        MAX_CONCURRENT_REQUESTS: 128
-        MAX_INPUT_LENGTH: 1024
-        MAX_TOTAL_TOKENS: 2048
+    baseImage: ghcr.io/huggingface/text-generation-inference:1.0.2
```

Comment:
We have a few different version tags in the config file. can we standardize on a single tag version for the tgi and tei ones? (they won't be the same tags between tei/tgi, but all tgi tags should be the same for example)